app.js

```diff
@@ -15,11 +15,11 @@ app.use(express.json());
 app.use("/api/contacts", contactsRouter);
 
 app.use((req, res) => {
-  res.status(404).json({ message: "404 - Not found" });
+  res.status(404).json({ message: "Not found" });
 });
 
 app.use((err, req, res, next) => {
-  res.status(500).json({ message: err.message });
+  res.status(err.status || 500).json({ message: err.message });
```

Comment:
potencjalnie mozesz dawac info na temat tego, co poszlo nie tak (nie zawsze chcemy udostepniac takie informacje, bo kazdy kto hituje serwer je dostanie). Nie mowie, ze masz to zmienic, ale "keep that in mind" :)

Reply:
Dzięki za info :)